src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -203,14 +203,18 @@ private boolean isTargetBranchVersionBehind(MergeReview mergeReview) {
 			return false;
 		}
 
-		// If no versions exist between the date range, then the target Branch is not a version behind.
 		CodeSystem codeSystem = codeSystemService.findClosestCodeSystemUsingAnyBranch(sourcePath, false);
-		String codeSystemShortName = "SNOMEDCT";
-		if (codeSystem != null) {
-			codeSystemShortName = codeSystem.getShortName();
+		if (codeSystem == null) {
+			return false;
+		}
+
+		Branch releaseBranch = sBranchService.findLatestReleaseBranchByCodeSystem(codeSystem);
+		if (releaseBranch == null) {
+			return false;
 		}
 
-		return !codeSystemService.findByImportDateBetweenRange(codeSystemShortName, leftTopLevelHeadTimestamp, rightTopLevelHeadTimestamp).isEmpty();
+		// If no versions exist between the date range, then the target Branch is not a version behind.
+		return !codeSystemService.findByImportDateBetweenRange(codeSystem.getShortName(), releaseBranch.getHeadTimestamp(), rightTopLevelHeadTimestamp).isEmpty();
```

Comment:
@dmcgihtsdo After use MAN's head time to look up the release branch (base's time), we don't need to check code system version. We can just compare these two dates.